macros/apply_regex.sql

```diff
@@ -0,0 +1,35 @@
+{%- macro apply_regex(column_name, regex) -%}
+
+    {{ return(adapter.dispatch('apply_regex')(column_name, regex)) }}
+
+{%- endmacro -%}
+
+{%- macro default__apply_regex(column_name, regex) -%}
+
+    regexp_like({{ column_name }}, '{{ regex}}')
+
+{%- endmacro -%}
+
+{%- macro snowflake__apply_regex(column_name, regex) -%}
+
+    regexp_like({{ column_name }}, '{{ regex }}')
+
+{%- endmacro -%}
+
+{%- macro bigquery__apply_regex(column_name, regex) -%}
+
+    regexp_contains(column_name, r'{{ regex }}')
```

Comment:
This column_name reference is missing brackets. Getting an error when testing in BigQuery.

![image](https://github.com/deepson-maitri/tuva/assets/106100509/63460ed0-68a5-4740-b5ec-1825dec2cced)


```suggestion
    regexp_contains({{ column_name }}, r'{{ regex }}')
```